packages/ngtools/webpack/src/ngcc_processor.ts

```diff
@@ -112,7 +113,12 @@ class NgccLogger implements Logger {
 
   debug(..._args: string[]) { }
 
-  info(..._args: string[]) { }
+  info(..._args: string[]) {
+    const msg = _args.join(' ');
+    if (!msg.includes(this.alreadyProcessMsg)) {
+      process.stderr.write(`\n${msg}\n`);
```

Comment:
What what use the standard output?

Reply:
Because it's a progress-like message. The webpack ProgressPlugin uses stderr for progress reporting, and so does Architect CLI. 

I remember a discussion with @clydin about this too where I was pretty convinced progress-like messages should go there.

Reply:
Thanks for the clarification